enos/modules/run_workloads/jobs/plugin-nfs-controllers.nomad.hcl

```diff
@@ -14,7 +14,7 @@ job "nfs-controller" {
       driver = "docker"
 
       config {
-        image = "registry.gitlab.com/rocketduck/csi-plugin-nfs:0.6.1"
+        image = "registry.gitlab.com/rocketduck/csi-plugin-nfs:latest"
```

Comment:
`:latest` is less than ideal long-term, because it could change in unexpected ways. I bet Florian would give us a tag here if we asked nicely :) https://gitlab.com/rocketduck/csi-plugin-nfs/container_registry/1601050

Reply:
Looks like he's got a pipeline running already! I'll update once that's landed.

Reply:
Done!